openapi/doc-ru-config-values.yaml

```
@@ -109,11 +109,14 @@ properties:
         type: object
         description: |
           Селектор разрешенных классов хранения, которые можно использовать для виртуальных образов на PVC.
+        minProperties: 1
```

```suggestion
        minProperties: 1
```
This an below dosn't make sense in the doc-ru specs